cluster/addons/cluster-monitoring/influxdb/heapster-controller.yaml

```diff
@@ -20,7 +20,7 @@ spec:
         kubernetes.io/cluster-service: "true"
     spec:
       containers:
-        - image: gcr.io/google_containers/heapster:v0.18.1
+        - image: thuctnguyen/heapster:v0.18.2-rc1
```

Comment:
I assume this change was unintentional. Kindly revert these changes. In general, once we move to the main heapster repo, these changes will not be necessary.